index.html

```diff
@@ -3951,41 +3951,191 @@ <h2>Syntaxes</h2>
+unique to the implementation approaches to data modeling, value usage, and
+parsing approaches.
+          </p>
+
+          <p>
+A JSON document becomes a [[!JSON-LD]] document when it includes an `@context`
+property with a URL associated with a context file. The context file provides
+the mapping between the idiosyncratic in-document terminology and universally
+shareable identifiers and value definitions. This provides not only
+contextualization and meaning, but also reusability through universally
+shareable terms and object-oriented structuring. This enables aggregation and
+collation of data without the need for additional upfront integration
+development time for each newly discovered JSON format. Future community
+credential definitions will focus on the use of their terminology rather than
+force implementers to parse another unique JSON format specific to that
+community or use case.
+          </p>
+
+          <p>
+Consequently, without any additional processing a JSON document gains the value
+of an in-message association with its terminology definitions. Details on
+processing VCDM documents consistently with the context file, but without
+[[!JSON-LD]] tooling can be found in the “JSON Processing” section below.
+          </p>
+
+          <p>
+The referenced context file provides the foundation for a consistent
+grammatical layer based on object-oriented modeling and community driven
+terminology selection which enables coexistence of various JSON tree shapes and
+colloquial terms while preventing likely terminology collisions or creating a
+need to write new tree traversal code for every new media type or JSON shape or
+parsing approach.
+          </p>
+
+          <p>
+Verifiable Credentials Data Model uses this consistent foundation to provide a
+common layer across all credentials, presentations, and transaction formats
+within a single, consistently parseable media type definition. This allows for
+ecosystem extensibility without requiring a centralized registry to be updated
+for every new application or community. The core data model terms are
+registered and specified while communities are free to build on that foundation
+through their own specification process. The terminology mappings are
+referenced from the VCDM document removing the need for an ever changing
+landscape of “private claims” or narrowly defined “public claims” housed in a
+centralized registry. The common grammar and object-oriented data modeling
+remain the same and properly contextualized terms can commingle. Applications
+and communities can easily extend what is claimed and signed through the use of
+vocabularies.
+          </p>
+
+          <p>
+The underpinning graph data model enables signing of the document’s statements
+rather than just the envelope it’s contained in. When using this feature, the
+data–while signed–is accessible in the same way it had been prior to signing
```

Comment:
```suggestion
The underpinning graph data model enables signing of the document's statements
rather than just the envelope in which it's contained. When using this feature, the
data — while signed — is accessible in the same way it had been prior to signing,
```